build.gradle

```diff
@@ -28,17 +28,11 @@ java {
 	withSourcesJar()
 }
 
-//tasks.withType(Javadoc) {
-//	// Set the source files for which Javadoc will be generated
-//	source = sourceSets.main.allJava
-//
-//	// Set the destination directory for the generated Javadoc
-//	destinationDir = file("$buildDir/docs/javadoc")
-//
-//	// Specify the files or packages to be excluded from Javadoc generation
-//	exclude '**/AsyncStateOptions.java'
-//	exclude '**/ProcessOptions.java'
-//}
+tasks.withType(Javadoc) {
```

Comment:
lol, what is the error? 
What I ran into before was some syntax error in the class comments that caused the java doc generation failure

Reply:
The javadoc task failed because it could not find the symbol AsyncStateOptionsBuilder from the lombok. But the real reason should be the same as what you described above, I didn't locate it for now. Once I started to revist the related codes for database,  I think I will find it and finally fix it.